src/plugins/commander/commands/rc.js

```diff
@@ -0,0 +1,23 @@
+const Command = require('../structs/Command.js');
+
+class RCCommand extends Command {
+    constructor(bot) {
```

Comment:
You should probably add FormatterPlugin to the dependencies, although I it wouldn't really throw any errors since it's not used in the constructor. Still something to keep consistent, you can look at parse.js for an example of using that dependency